agent/src/test/java/com/microsoft/hydralab/agent/runner/ActionExecutorTest.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package com.microsoft.hydralab.agent.runner;
+
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import com.microsoft.hydralab.agent.test.BaseTest;
+import com.microsoft.hydralab.common.entity.common.DeviceAction;
+import com.microsoft.hydralab.common.entity.common.DeviceInfo;
+import com.microsoft.hydralab.common.entity.common.Message;
+import com.microsoft.hydralab.common.util.SerializeUtil;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+class ActionExecutorTest extends BaseTest {
+
+    ActionExecutor actionExecutor = new ActionExecutor();
+
+    @Test
+    void doAction() {
```

Comment:
Thank you for adding this UT! For the naming convention of a UT method, we can learn from: [How to Choose the Best Unit Test Method Naming Convention](https://methodpoet.com/unit-test-method-naming-convention/)